src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -120,6 +120,17 @@ std::string postEntities
 
     TIMED_RENDER(answer = oe.render(ciP, ""));
   }
+  else if (rhttpcode == SccInvalidParameter)
+  {
+    // This v1 -> v2 error conversion is a little crazy (for example, the same
+    // SccInvalidParameter at v1 level could match two different error conditions
+    // at v2 level, making harder de logic). However, I'm afraid that we need
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/a8d0650bc335b0e735f4c57a87096bbbede7507b